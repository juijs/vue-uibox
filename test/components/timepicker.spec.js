import { mount } from 'vue-test-utils'
import UixTimepicker from '@/components/timepicker'

describe('timepicker.js', () => {
    const wrapper = mount(UixTimepicker, {
        propsData: {
            year: 2015,
            month: 11,
            date: 2,
            hours: 6,
            minutes: 20,
            useCalendar: true
        }
    });

    it('snapshot!!', () => {
        wrapper.vm.$nextTick(() => {
            expect(wrapper.vm.$el).toMatchSnapshot();
        });
    });
})