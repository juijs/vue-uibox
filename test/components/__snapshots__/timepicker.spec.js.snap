// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`timepicker.js snapshot!! 1`] = `
<div
  class="timepicker-group"
>
  <div
    class="timepicker calendar small"
  >
    <input
      class="year"
      data-value="2015"
      maxlength="4"
      type="input"
    />
    -
    <input
      class="month"
      data-value="11"
      maxlength="2"
      type="input"
    />
    -
    <input
      class="date"
      data-value="2"
      maxlength="2"
      type="input"
    />
     
    <i
      class="icon-calendar"
      style="cursor: pointer;"
    />
  </div>
   
  <div
    class="timepicker small"
  >
    <input
      class="hours"
      data-value="6"
      maxlength="2"
      type="input"
    />
     :
        
    <input
      class="minutes"
      data-value="20"
      maxlength="2"
      type="input"
    />
     
    <i
      class="icon-arrow7"
    />
  </div>
   
  <div
    class="datepicker"
  >
    <div
      class="head"
    >
      <div
        class="prev"
      >
        <i
          class="icon-chevron-left"
        />
      </div>
       
      <div
        class="title"
      >
        2015.11
      </div>
       
      <div
        class="next"
      >
        <i
          class="icon-chevron-right"
        />
      </div>
    </div>
     
    <table
      class="body"
    >
      <thead>
        <tr>
          <th>
            SU
          </th>
          <th>
            MO
          </th>
          <th>
            TU
          </th>
          <th>
            WE
          </th>
          <th>
            TH
          </th>
          <th>
            FR
          </th>
          <th>
            SA
          </th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>
            1
          </td>
          <td
            class="active"
          >
            2
          </td>
          <td>
            3
          </td>
          <td>
            4
          </td>
          <td>
            5
          </td>
          <td>
            6
          </td>
          <td>
            7
          </td>
        </tr>
        <tr>
          <td>
            8
          </td>
          <td>
            9
          </td>
          <td>
            10
          </td>
          <td>
            11
          </td>
          <td>
            12
          </td>
          <td>
            13
          </td>
          <td>
            14
          </td>
        </tr>
        <tr>
          <td>
            15
          </td>
          <td>
            16
          </td>
          <td>
            17
          </td>
          <td>
            18
          </td>
          <td>
            19
          </td>
          <td>
            20
          </td>
          <td>
            21
          </td>
        </tr>
        <tr>
          <td>
            22
          </td>
          <td>
            23
          </td>
          <td>
            24
          </td>
          <td>
            25
          </td>
          <td>
            26
          </td>
          <td>
            27
          </td>
          <td>
            28
          </td>
        </tr>
        <tr>
          <td>
            29
          </td>
          <td>
            30
          </td>
          <td
            class="none"
          >
            1
          </td>
          <td
            class="none"
          >
            2
          </td>
          <td
            class="none"
          >
            3
          </td>
          <td
            class="none"
          >
            4
          </td>
          <td
            class="none"
          >
            5
          </td>
        </tr>
        <tr>
          <td
            class="none"
          >
            6
          </td>
          <td
            class="none"
          >
            7
          </td>
          <td
            class="none"
          >
            8
          </td>
          <td
            class="none"
          >
            9
          </td>
          <td
            class="none"
          >
            10
          </td>
          <td
            class="none"
          >
            11
          </td>
          <td
            class="none"
          >
            12
          </td>
        </tr>
      </tbody>
    </table>
  </div>
  <div
    style="top: 0px; position: absolute; width: 0px; height: 0px; cursor: pointer; right: 2px;"
  />
  <div
    style="top: 0px; position: absolute; width: 0px; height: 0px; cursor: pointer; right: 2px;"
  />
</div>
`;
